### PR TITLE
Flash message on error

### LINKED ISSUE #12: Flash message on error
[TODO here.](https://github.com/osuosl/timesync-frontend-flask/blob/59316cb6ed22b81794234f06648fca6b753e237c/app/views.py#L100)

[and here](https://github.com/osuosl/timesync-frontend-flask/blob/8b95a4ce2f8adf4eac8bb9cd12275a06adf6e77d/app/views.py#L129)

We also need better error handling on the methods that call `getUser()`, [see here](https://github.com/osuosl/timesync-frontend-flask/blob/develop/app/views.py#L52).

### PR DESCRIPTION
**Pull request by MaraJade**
Fixes #12 
## Changes in this PR.

- [X] Errors are now flashed instead of returned
- [X] Error messages now print out the complete error and text
## Testing this PR.

1. Run `nosetests`
### Expected Output.

```
$ nosetests
............................
----------------------------------------------------------------------
Ran 28 tests in 0.534s

OK
```

@osuosl/devs

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -83,12 +97,12 @@ def login():
 
         # If regular error, tell user error
         if 'error' in token:
-            flash(token['text'])
+            flash("Error: " + token['error'] + " - " + token['text'])
```
Comment (Kennric) [inline app/views.py]: Is there a reason why we flash here, but use the error method elsewhere?

**Comment by pop** [comment]
lgtm +1

File: app/views.py [hunk 2]
```diff
@@ -82,13 +96,12 @@ def login():
...
 
         # If regular error, tell user error
+        error_message(token)
         if 'error' in token:
-            flash(token['text'])
             status = token['status']
-        # If pymesync error, tell user vague error
+        # If pymesync error, tell user error
         elif 'pymesync error' in token:
             print token
```
Comment (Kennric) [inline app/views.py]: Why are we printing here? Actually, is this whole elseif necessary? Seems like error_message() will take care of printing errors, and this block isn't doing anything else.

**Comment by aaroncohen73** [comment]
@MaraJade In Pymesync 0.1.5, the get_\* methods were changed to always return lists, even if there is only one item in the list or if an error was returned. As a result of this change, some places in your code (Such as line 74 in `app/views/create_time.py`) don't catch errors correctly because the `error_message` function is getting a list instead of a dictionary.

**Comment by aaroncohen73** [comment]
lgtm +1

**Comment by Kennric** [comment]
+1

**Pull request merged**